python/requirements/dev.txt

```diff
@@ -30,5 +30,5 @@ beautifulsoup4==4.6.0
 inflection==0.3.1
 pyOpenSSL==17.5.0
 pytest==3.6.1
-lxml==3.8.0
+lxml==4.2.5
```

Comment:
this really needs to be kept in sync with the version in the other repository

Reply:
more to the point, why do you need this?

Reply:
I will remove this